Corona-Warn-App/src/main/res/layout/event_registration_category_fragment.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/category_root"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:contentDescription="@string/tracelocation_organizer_category_title">
+
+        <include
+            android:id="@+id/category_navbar"
+            layout="@layout/include_header"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            app:icon="@{@drawable/ic_close}"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:title="@{@string/tracelocation_organizer_category_title}" />
+
+        <ScrollView
```

Comment:
Why not a `RecyclerView` ? I think you can use the generic item headers and items.

Reply:
Hi! I thought about a `RecyclerView` but then decided on this approach, as I think it is simpler. Why would you prefer a `RecyclerView`?

Reply:
RecyclerView recycles the views. the more there are item, the more the performance will be affected. you assume that we have only those types, but we might need more types in the futures with different categories. in my opinion this approach is not flexible for future changes while RecyclerView with a generic adapter would be a good fit. why did you decide against it?

Reply:
Since the list is very small and simple and most items fit on a single screen on most phones IMO performance is not really an issue here. 

I decided for this approach as I think it is a bit simpler, but your argument about maintainability convinced me to implement it as `RecyclerView`. Will do this in an upcoming commit.

Reply:
Refactored to `RecyclerView` in new commit.